bgpd/bgp_routemap.c

```diff
@@ -2410,10 +2412,14 @@ route_set_aspath_replace(void *rule, const struct prefix *dummy, void *object)
 	as_t configured_asn;
 	char *buf;
 	char src_asn[ASN_STRING_MAX_SIZE];
+	char acl_list_name[ACL_NAME_MAX_SIZE];
+	uint32_t acl_list_name_len = 0;
+	char *buf_acl_name = NULL;
+	static const char asp_acl[] = "as-path-access-list";
+	struct as_list *aspath_acl = NULL;
 
 	if (path->peer->sort != BGP_PEER_EBGP) {
-		zlog_warn(
-			"`set as-path replace` is supported only for EBGP peers");
+		zlog_warn("%s is supported only for EBGP peers", __func__);
```

Comment:
This change is non-user-readable for the end-user, IMO. Why an old is bad?

Reply:
taken into account